src/openrct2/ride/RideData.h

```diff
@@ -324,6 +324,93 @@ struct TrackDrawerDescriptor
     bool HasCoveredPieces() const;
 };
 
+enum class RtdFlag : uint8_t
+{
+    hasTrackColourMain = 0,
+    hasTrackColourAdditional = 1,
+    hasTrackColourSupports = 2,
+    hasSinglePieceStation = 3, // Set by flat rides, tower rides and shops/stalls.
+    hasLeaveWhenAnotherVehicleArrivesAtStation = 4,
+    canSynchroniseWithAdjacentStations = 5,
+    trackMustBeOnWater = 6, // used only by boat Hire and submarine ride
+    hasGForces = 7,
+    cannotHaveGaps = 8, // used by rides that can't have gaps, like those with a vertical tower, such
+    // as the observation tower
+    hasDataLogging = 9,
+    hasDrops = 10,
+    noTestMode = 11,
+    hasCoveredPieces = 12, // set on rides with two varieties,
+    // like the u and o shapes of the dinghy slide
+    // and the dry and submerged track of the water
+    // coaster
```

Comment:
This seems oddly indented spaced, like some of the other comments here. Could you look into this?